mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -4,9 +4,9 @@
 #         Stefan Appelhoff <stefan.appelhoff@mailbox.org>
 #
 # License: BSD (3-clause)
-
 import os.path as op
 from os import unlink
+from collections import OrderedDict as odict
```

Comment:
avoid changing names of builtin objects. It makes your code harder to read. I hard to scroll the file to understand the code below